test/spider_html_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule ParseTest do
+  use ExUnit.Case
+
+  @urls [
+    "http://onigumo.local/hello.html",
+    "http://onigumo.local/bye.html"
+  ]
+  @html ~s(<!doctype html>
+<html>
+    <body>
+        <section id="content">
+            <p class="headline">Floki</p>
+            <a href="http://onigumo.local/hello.html">Hello</a>
+            <a href="http://onigumo.local/bye.html">Bye</a>
+            <a id="b"></a>
+            <span data-model="user">onigumo</span>
```

Comment:
We could utilize a test verifying that a non “a” tag with a “href” attribute does not appear in the result. Such tag is not valid HTML, it can still appear in the real world, though. We can discuss, whether finding hrefs on other tags should be in the result, but the current implementation doesn’t do so, so it needs have a test. See #170.

Reply:
My previous claim was false: there are more elements that can have the _href_ attribute: _area_, _link_ etc.